lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -200,17 +215,20 @@ class LineChartPainter extends AxisChartPainter<LineChartData> with TouchHandler
   /// first one is the sharp corners line on spot connections
   /// second one is curved corners line on spot connections,
   /// and we use isCurved to find out how we should generate it,
-  Path _generateBarPath(Size viewSize, LineChartBarData barData) {
+  Path _generateBarPath(Size viewSize, LineChartBarData barData, Path appendToPath) {
```

Comment:
May I ask you to explain the usage of `appendToPath`?

Reply:
I added some documentation:

`If you want to concatenate paths together for creating an area between multiple bars for example, you can pass the appendToPath`

Is that better? Maybe you know a better way to concatenate path's?